src/Microsoft.AspNetCore.WebHooks.Receivers/Filters/WebHookVerifySignatureFilter.cs

```diff
@@ -61,6 +61,41 @@ public bool IsApplicable(string receiverName)
             return string.Equals(ReceiverName, receiverName, StringComparison.OrdinalIgnoreCase);
         }
 
+        /// <summary>
+        /// Converts a base64-encoded string to a <see cref="T:byte[]"/> (with or without URI-safe mode encoding).
+        /// </summary>
+        /// <param name="content">The Base64-encoded string to convert.</param>
+        /// <returns>The converted <see cref="T:byte[]"/>.</returns>
+        public static byte[] FromBase64(string content)
```

Comment:
Doesn't web utilities have this?

Reply:
Turns out I believed the comments and shouldn't have. `Base64UrlTextEncoder` doc comments say it can't decode a string containing padding. But, it's fine either way 👍